### ISSUE TITLE
Batch Norm Layer Fusing for Transposed Convolution

### ISSUE THREAD

**Issue by Coke-das-JHL** [Xilinx/brevitas#484]
How to perform Batch Norm Layer Fusing for Transposed Convolution?

Or, can you advise me on another way for Transposed Convolution with Batch Normalization?

Thank you

**Comment by MohamedA95** [comment]
Check [this](https://github.com/pytorch/pytorch/blob/2f8c0cb2a44211428f5f31a5e04a861e952d3a0e/torch/nn/utils/fusion.py#L16) function it's used, after training.

**Comment by Coke-das-JHL** [comment]
> Check [this](https://github.com/pytorch/pytorch/blob/2f8c0cb2a44211428f5f31a5e04a861e952d3a0e/torch/nn/utils/fusion.py#L16) function it's used, after training.

Thank you!
